src/lib/Microsoft.Health.Fhir.R4.Ingest/Template/QuantityFhirValueProcessor.cs

```diff
@@ -20,9 +21,21 @@ protected override DataType CreateValueImpl(QuantityFhirValueType template, IObs
             EnsureArg.IsNotNull(inValue, nameof(inValue));
             IEnumerable<(DateTime, string)> values = EnsureArg.IsNotNull(inValue.Data, nameof(IObservationData.Data));
 
+            decimal value;
+            try
+            {
+                value = decimal.Parse(values.Single().Item2, CultureInfo.InvariantCulture);

```

Comment:
Let me know if there are multiple values then if that should be considered a non handleable situation.

Reply:
Currently we throw an exception I believe, so handling that scenario as part of this change seems appropriate. 

We do have an open issue https://github.com/microsoft/iomt-fhir/issues/82 to allow different strategies but outside the scope of this PR.